internal/kibana/reconciler_test.go

```diff
@@ -310,16 +311,12 @@ func TestDeploymentDifferentWithKibanaEnvVar(t *testing.T) {
 		clusterRequest.cluster.Namespace,
 		"kibana",
 		"kibana",
+		1,
```

Comment:
i realize this is for a test, but can we avoid hard coding values?

Reply:
Done